skyportal/models.py

```diff
@@ -206,6 +208,71 @@ def desi_dr8_url(self):
         return (f"http://legacysurvey.org/viewer/jpeg-cutout?ra={self.ra}"
                 f"&dec={self.dec}&size=200&layer=dr8&pixscale=0.262&bands=grz")
 
+    def airmass(self, telescope, time, below_horizon=np.inf):
+        """Return the airmass of the Obj at time `time` from Telescope
+        `telescope`.
+
+        Parameters
+        ----------
+
+        telescope: skyportal.models.Telescope
+            The telescope to use for the airmass calculation
+
+        time: astropy.time.Time
+            The time or times at which to calculate the airmass
```

Comment:
If this can be times (plural), then the type needs to be astropy.time.Time or list of Time